test/BUILD.gn

```diff
@@ -348,8 +349,13 @@ test("brave_unit_tests") {
       "//brave/chromium_src/components/translate/core/browser/translate_manager_unittest.cc",
       "//brave/components/brave_shields/browser/brave_shields_p3a_unittest.cc",
       "//brave/components/brave_shields/browser/brave_shields_util_unittest.cc",
-      "//brave/components/omnibox/browser/fake_autocomplete_provider_client.cc",
-      "//brave/components/omnibox/browser/fake_autocomplete_provider_client.h",
+      "//brave/components/omnibox/browser/brave_bookmark_provider_unittest.cc",
```

Comment:
One more thing :) 
I recommend to use specific omnibox unit_tests target instead of adding each test file to this grand BUILD.gn.
I think we are going that way. - (Ex, `//brave/components/brave_sync:unit_tests`)

Reply:
I had a little bit of a play around with this. Do I need to migrate the whole `sources.gni` file to a `BUILD.gn` one?

Reply:
Just including test related files in new test target with proper deps list that used by test codes.

Reply:
:+1: